.ci/releasenotes-gen-xdoc-push.sh

```diff
@@ -51,7 +51,17 @@ java -jar contribution/releasenotes-builder/target/releasenotes-builder-1.0-all.
      -githubAuthToken "$READ_ONLY_TOKEN" \
      -generateXdoc \
      -xdocTemplate $BUILDER_RESOURCE_DIR/templates/xdoc_freemarker.template \
-     -publishXdoc -publishXdocWithPush
 
-echo "releasenotes.xml after commit:"
-head "checkstyle/src/xdocs/releasenotes.xml" -n 100
+cd ../
+
+CS_RELEASE_VERSION=$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
+              -Dexec.args='${project.version}' \
+              --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec | sed 's/-SNAPSHOT//')
+  
+LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                       | jq ".tag_name")
+
+mvn -e --no-transfer-progress versions:set -DgroupId=com.puppycrawl.tools -DartifactId=checkstyle \
+  -DoldVersion=10.4-SNAPSHOT -DnewVersion="$CS_RELEASE_VERSION"
+
```

Comment:
Remove this bump,.
Version bump is done by release.sh during mvn release:perform.

All you need in this PR is just update xdoc and commit and push it.


Please share prove of how it works in your repo.

Reply:
@romani I am a little confused, then what should the commit contain ( if not the change of project version)?

Reply:
It should contain xdoc update - release notes

Reply:
Example https://github.com/checkstyle/checkstyle/commit/47e6e280027940ba741d552ab9147fba7b4152a7